src/Storage/Storage.Autorest/docs/Get-AzStorageSku.md

```diff
@@ -0,0 +1,90 @@
+---
+external help file:
+Module Name: Az.Storage
+online version: https://learn.microsoft.com/powershell/module/az.storage/get-azstoragesku
+schema: 2.0.0
+---
+
+# Get-AzStorageSku
+
+## SYNOPSIS
+Lists the available SKUs supported by Microsoft.Storage for given subscription.
+
+## SYNTAX
+
+```
+Get-AzStorageSku [-SubscriptionId <String[]>] [-DefaultProfile <PSObject>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Lists the available SKUs supported by Microsoft.Storage for given subscription.
+
+## EXAMPLES
+
+### Example 1: List available Storage SKUs for a subscription 
+```powershell
+Get-AzStorageSku
+```
+
+```output
+Kind             Name            Tier     Location             ResourceType
+----             ----            ----     --------             ------------
+BlockBlobStorage Premium_LRS     Premium  {australiacentral}   storageAccounts
```

Comment:
It would better to add available zone (locationInfo.zones) to the default output format.
However, it there's no easy way to do it, we can give an example to do it.